galaxy.yml

```diff
@@ -10,7 +10,7 @@ namespace: pulp
 name: pulp_installer
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 3.6.4
+version: 3.6.5
```

Comment:
I'm not familiar with their versioning, do they have post releases?

Reply:
We've only done non-semantic strings like "3.6.5-1" and "3.6.5-2", and those were for new pulp_installer releases that installed to the existing pulpcore releases (like 3.6.5). I settled on that format after looking at other collections as examples.

I say we try it and see what happens. Make sure that galaxy recognizes it.